.github/workflows/tests.yml

```diff
@@ -0,0 +1,105 @@
+name: tests
+on:
+  push:
+    branches-ignore: [2.0]
+  workflow_dispatch:
+
+jobs:
+  test-browser:
+    runs-on: ubuntu-latest
+    env:
+      SF_USERNAME: ${{ secrets.SF_USERNAME }}
```

Comment:
these tests need another workflow that will
auth to the hub (like the NUTs do)
create a scratch org (json output will have username, instanceUrl)
`user password generate` to get the password

Reply:
then all the 7 tests can run in parallel with isolated orgs